### DISCUSSION TITLE
Plausible analytics container stops a few seconds after starting

### DISCUSSION THREAD

**Discussion by grsoares21** [plausible/analytics#832] [Self-Hosted Support]
Hello. I'm trying to run a self-hosted version of Plausible. I followed the [installation guide on the docs](https://plausible.io/docs/self-hosting) but the container that runs the Elixir app is being stopped a few seconds after being started.

Here's how the commands go:
![image](https://user-images.githubusercontent.com/6013294/110914919-4fb9df80-8317-11eb-8a47-d106963cfd7c.png)

I'm not experienced with either Docker nor Elixir so I didn't know how to get more details out of it. I'd appreciate any help in troubleshooting this. Please let me know how can I get any more details that might be needed.

Thanks in advance!

EDIT. I've just tried in a brand new server and it work. It's probably something to do with what I've already configured in this server I imagine, (e.g. I've already had nginx installed and configure, is it possible that it's interfering with the connection?).

**Comment by metmarkosaric** [comment]
thanks for the update @grsoares21! enjoy Plausible!
